my-app/src/components/Medicines/AddData.js

```diff
@@ -1,82 +1,99 @@
-import React, { Component } from 'react';
-
-
-
-const DEFAULT_STATE = {
-  id: "",
-  name: "",
-  type: "",
-  dosage: "",
-  date: "",
-  doctor: "",
-  other: "",
+import React, { useState, useEffect } from 'react';
+import firebase from '../Firebase/firebase'
+import { Button, Header, Modal } from 'semantic-ui-react'
+import { useTranslation } from 'react-i18next';
+import i18next from 'i18next';
+
+
+const AddData = () => {
+  const [data, setData] = useState('');
```

Comment:
please check previous comments

Reply:
I have errors when i try to use useReducer